src/allmydata/test/test_eliotutil.py

```diff
@@ -281,3 +282,75 @@ def f():
                 ),
             ),
         )
+
```

Comment:
The case where logged objects are _not_ JSON serializable are not tested at the moment. So you'd need to add tests for that for both args and kwargs (could be a single test that covers both).

Reply:
Right